app/views/projects/_form.html.erb

```diff
@@ -54,6 +54,14 @@
       </div>
     </div>
 
+    <div class="form-group">
+      <div class="col-lg-offset-2 col-lg-4">
+        <%= form.check_box :deploy_to_new_pods %>
+        <%= form.label :deploy_to_new_pods, 'Deploy to New Pods'%>
+        <span class="help-block">This a pod-specific service, and should be deployed to each pod, including new pods.</span>
```

Comment:
Nothing says deploy_groups have to be pods. We have deploy groups that aren't pods.